Sex max-width to vaadin-grid-filter

samiheikki opened the pull request:
Fixes #1176

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/vaadin/vaadin-grid/1398)
<!-- Reviewable:end -->

tomivirkki commented:
*[a discussion](https://reviewable.io/reviews/vaadin/vaadin-grid/1398#-LMHD_md45OG4HRzNPL8:-LMHD_meClzrXBlj85U4:bl7srmg) (no related file):*
Could use a test.
And the demos should be updated: note that there are some filters there that use a custom input that this fix doesn't affect.

---


<!-- Sent from Reviewable.io -->

moderndeveloperllc commented:
@samiheikki Might want to check that issue title. I don't think you like `vaadin-grid-filter` _that_ much 🤔

samiheikki commented:
*[a discussion](https://reviewable.io/reviews/vaadin/vaadin-grid/1398#-LMHD_md45OG4HRzNPL8:-LMbIh2S9ly4soXzNKUK:b4c9lyr) (no related file):*
<details><summary><i>Previously, tomivirkki (Tomi Virkki) wrote…</i></summary><blockquote>

Oh, sorry wasn't clear about the demos. Added `max-width` to the wrapped text fields (and changed the commit message ;) )
</blockquote></details>

:thumbsup: 

---


<!-- Sent from Reviewable.io -->
